rc/lsp.kak

```diff
@@ -1158,6 +1158,39 @@ incomingOrOutgoing = $1
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
+
+define-command lsp-inlay-hints -docstring "lsp-inlay-hints: Request inlay hints (lsp)" %{
```

Comment:
I think we should drop the `(lsp)` suffix because it will become obsolete once we remove the rust-analyzer variant.
I also prefer lowercase after colon (like most Kakoune commands) but that's an unrelated change.

```suggestion
define-command lsp-inlay-hints -docstring "lsp-inlay-hints: request inlay hints" %{
```